donate/settings/secure.py

```diff
@@ -0,0 +1,38 @@
+from .environment import env
+
+
+class Secure(object):
+    DEBUG = False
+    USE_X_FORWARDED_HOST = True
+    CSRF_COOKIE_SECURE = True
+    SESSION_COOKIE_SECURE = True
+    SECURE_BROWSER_XSS_FILTER = True
+    SECURE_CONTENT_TYPE_NOSNIFF = True
+    SECURE_HSTS_INCLUDE_SUBDOMAINS = True
+    SECURE_HSTS_SECONDS = 60 * 60 * 24 * 31 * 6
```

Comment:
I think the recommended value is 1 year? (also, fun aside: six months is not 186 days =D)